<launch>
    <!-- <arg name="republish_compressed_image" default="1"/> -->
    <!-- <arg name="numPointsPerVertArc" default="5"/>
    <arg name="numVertArc" default="6"/>
    <arg name="numRadius" default="2"/>
    <arg name="pointCloudOnly" default="0"/>

    <group if="$(eval arg('pointCloudOnly') == 0)">
        <node name="trajectory_planner_node" pkg="gripper_blaser_ros" type="trajectory_planner_node"  
        output="screen" args = "1 $(arg numPointsPerVertArc) $(arg numVertArc) $(arg numRadius)"/>
    </group> -->
    
    <!-- <env name="GMON_OUT_PREFIX" value="gripper_blaser_node" /> -->
    <node name="gripper_blaser_node"  
        pkg="gripper_blaser_ros" type="gripper_blaser_node" output="screen" 
        args = "$(dirname)/../cfg/ximea_80_calib.yaml"/>

    <rosparam file="$(dirname)/../cfg/calib_params.yaml"/>
    <rosparam file="$(dirname)/../cfg/stitcher_params.yaml"/>


    <!-- Republish compressed image as raw image
    <group if="$(eval arg('republish_compressed_image') == 1)">
        <node name="visual_republish" pkg="image_transport" type="republish" args="in:=/ximea_cam/image_raw/compressed compressed out:=/fujun"/>
    </group>  -->

    <!-- rviz -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find gripper_blaser_ros)/cfg/rviz_config_demo.rviz"/>
</launch>
